packages/gatsby-script/src/request-idle-callback-shim.ts

```diff
@@ -0,0 +1,18 @@
+// https://developer.chrome.com/blog/using-requestidlecallback/#checking-for-requestidlecallback
+// https://github.com/vercel/next.js/blob/canary/packages/next/client/request-idle-callback.ts
+
+export const requestIdleCallback =
```

Comment:
Why is this not imported in `gatsby-script`?

Reply:
Ouch! probably missed a commit while cherry-picking.